docs/development.md

```diff
@@ -51,8 +51,8 @@
    - [install instructions][kustomizelinux] on Linux
 6. Configure Python 2.7+ with [pyenv][pyenv] if your default is Python 3.x.
 7. Install make.
-
-[go]: https://golang.org/doc/install
+8. Install [timeout][timeout]
+   - `brew install coreutils && sudo ln -s /usr/local/bin/gtimeout /usr/local/bin/timeout` on MacOS.
```

Comment:
@ksubrmnn @sozercan do either of you know if we need equivalent instructions for windows wsl?

Reply:
it's already in `/usr/bin/timeout` in WSL for me

Reply:
if you meant the whole instructions, using Linux should be fine in WSL